rts/System/Platform/Misc.cpp

```diff
@@ -519,4 +519,20 @@ std::string ExecuteProcess(const std::string& file, std::vector<std::string> arg
 	return execError;
 }
 
+void OpenURL(const std::string& url)
```

Comment:
url should be parsed for beeing a valid url:

i.e. that things like rm -rf / aren't possible.